spec/jobs/index_full_text_content_job_spec.rb

```diff
@@ -5,10 +5,16 @@
 RSpec.describe IndexFullTextContentJob do
   describe '#perform' do
     it 'adds content to solr' do
-      allow(Search).to receive(:client).and_return(instance_double(RSolr::Client, add: nil))
+      allow(Search).to receive(:client).and_return(instance_double(RSolr::Client, update: nil))
       allow(PurlObject).to receive(:new).and_return(instance_double(PurlObject, to_solr: [{ id: 1 }]))
```

Comment:
Not a huge deal because this is all stub data (and feel free to ignore) but I wonder if we should add `druid: 'x'`) to the stub PurlObject to make it clearer that there is a connection to the delete that happens before.